### ISSUE TITLE
[Usable - Overview]: In Browse mode, NVDA announce irrelevant information 'Clickable' repeatedly on graph below 'Behavior impact on compromise rate'

### ISSUE THREAD

**Issue by Shubham786786** [microsoft/fluentui#17072]
**User Experience:**
Screen Reader user will be impacted here as NVDA announcing irrelevant information repeatedly.

**Test Environment:**
OS build Version 2004 (OS Build 21301.1000)
Browser Version: Microsoft Edge Dev Version 90.0.796.0
Screen Reader: NVDA 2020.4
URL: [https://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/heads/master/charting/dist/index.html#/examples/areachart](url) 

**Repro Steps:**
1. Open above URL on Anaheim Dev Browser and login with valid credentials.
2. Open NVDA.
3. Navigate to ‘Overview’ tab item.
4. Navigate to graph below 'Behavior impact on compromise rate' in browse mode.
5. Observe the issue.

**Actual Result:**
NVDA announce irrelevant information 'Clickable' repeatedly on graph below 'Behavior impact on compromise rate' user navigate in browse mode, NVDA pronounce 'Graphic Clickable Clickable Clickable ...... '.

**Expected Result:**
NVDA Should not announce irrelevant information 'Clickable' repeatedly on graph below 'Behavior impact on compromise rate' user navigate in browse mode.

**Observation:**
1. Issue does not repro with Narrator as its remain silent.

**Guideline reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7bF98F4235-B6BD-47D3-8DA4-F28B9B6FBFA8%7d&file=Guideline%20101%20%E2%80%93%20Recommended%20Assistive%20Technology%20Tools%20for%20MAS%20Testing%20of%20Microsoft%20Digital%20Properties.docx&action=default&mobileredirect=true&cid=e76f6c4f-0f24-49a4-b4a4-095b9398d8e8](url)

**Attachment**

[Usable-NVDA starts announcing the clickable information multiple times.zip](https://github.com/microsoft/fluentui/files/6008595/Usable-NVDA.starts.announcing.the.clickable.information.multiple.times.zip)

**Comment by shivasai09** [comment]
filed the bug at nvda to get more context
https://github.com/nvaccess/nvda/issues/12090

**Comment by gouttierre** [comment]
@shivasai09  - Thank you for the feedback into the issue you are experiencing 

@khmakoto - Would you be able to confirm if this is a regression, or if this behavior is an issue with that particular browser and NVDA?
cc @smhigley
